include/Makefile

```diff
@@ -6,8 +6,7 @@ HEADERS = $(wildcard *.h) $(foreach dir, machine sys, $(wildcard $(dir)/*.h))
 INSTALL-FILES = $(addprefix $(SYSROOT)/usr/include/, $(HEADERS))
 
 SOURCES_H = $(foreach dir, . dev mips sys, $(wildcard $(dir)/*.h))
-FORMAT-EXCLUDE = mips/asm.h mips/m32c0.h \
-                 sys/syscall.h sys/syscallargs.h
+FORMAT-EXCLUDE = mips/m32c0.h sys/syscall.h sys/syscallargs.h
 
```

Comment:
Maybe you could move reformatting of `include/mips/asm.h` to a different PR?

Reply:
Did that on my own. Please clean up `mips/asm.h` header file.